test/stress/s3benchmark.cc

```diff
@@ -0,0 +1,225 @@
+#include "upload_s3.h"
+
+#include <iostream>
```

Comment:
If possible, please avoid streams for consistency with the rest of the code base.  In util/posix.h you can find useful helper functions to write into files (`SafeWrite`).